gridcomps/ExtData2G/ExtDataGridCompNG.F90

```diff
@@ -108,7 +109,7 @@ MODULE MAPL_ExtDataGridComp2G
                                           !! are not actually required
      type(ESMF_Config)    :: CF
      logical              :: active = .true.
-     logical              :: file_weights = .false.
+     logical              :: file_weights = .true.
```

Comment:
Do we want this on all the time?

Reply:
@mathomp4 this is at least debatable, although I'm surprised Bill had this given the issues with this I still have not been able to reproduce with the reading/writing of the weights at "Bill" scale